src/test/scala/com/mwz/sonar/scala/util/syntax/SonarConfigSpec.scala

```diff
@@ -31,12 +31,13 @@ class SonarConfigSpec extends FlatSpec with Matchers {
     val conf = new MapSettings()
       .setProperty("path", "this/is/a/path, another/path")
       .asConfig()
+    val defaultPath = List(Paths.get("default/path"), Paths.get("default/path2"))
```

Comment:
Just one minor thing, this should be called `defaultPaths` _(since now there are many)_.

Reply:
yes of course, I'll fix that